### ISSUE TITLE
 INSTALLATION FAILED: rendered manifests contain a resource that already exists

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1448]
I've been seeing this occasionally for the past few days during cluster setup.  Here's the log:

```
*** STATUS: setup: neon-system-db
START: sudo bash -c 'mkdir -p /var/local/neonkube/setup'
END [OK]
START: sudo bash -c 'if [ -f "/var/local/neonkube/setup/helm-unzip" ] ; then exit 0; else exit 1; fi'
END [OK]
START: sudo bash -c 'mkdir -p /var/local/neonkube/setup'
END [OK]
START: sudo bash -c 'if [ -f "/var/local/neonkube/setup/helm-install-neon-system-db" ] ; then exit 0; else exit 1; fi'
END [ERROR=1]
*** STATUS: helm install: neon-system-db
----------------------------------------
START-BUNDLE: ./script.sh
*** START TEXT FILE: script.sh

set -euo pipefail

cd /lib/neonkube/helm

helm install neon-system-db --namespace neon-system -f postgres_operator/values.yaml --set priorityClassName=neon-data --set metrics.enabled=True --set-string metrics.interval="1m" --set-string resources.requests.memory="128Mi" --set-string resources.limits.memory="256Mi" --set replicas=1 --set-string podPriorityClassName="neon-data" ./postgres_operator

START=`date +%s`
DEPLOY_END=$((START+15))

set +e

until [ `helm status neon-system-db --namespace neon-system | grep "STATUS: deployed" | wc -l` -eq 1  ];
do
  if [ $((`date +%s`)) -gt $DEPLOY_END ]; then
    helm delete neon-system-db || true
    exit 1
  fi
   sleep 1
done
*** END TEXT FILE
*** START TEXT FILE: __run.sh
#!/bin/sh

chmod 700 "script.sh"
./script.sh
*** END TEXT FILE
*** Uploading: /home/sysadmin/.neon/exec/501a5731-f099-4dbb-aaf5-533687b10712/__bundle.zip
START: sudo bash -c 'if [ -f "/home/sysadmin/.neon/upload/__bundle.zip-c2924764-cca8-4d82-aa4b-bd8f8459cf75" ] ; then rm "/home/sysadmin/.neon/upload/__bundle.zip-c2924764-cca8-4d82-aa4b-bd8f8459cf75" ; fi'
END [OK]
STDERR
    Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: PriorityClass "neon-data" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "neon-system-db"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "neon-system"
END [ERROR=1]
END-BUNDLE
----------------------------------------
```

**Comment by jefflill** [comment]
So the Helm chart installing **neon-system-db** is failing because it appears to be trying to create the **neon-data** priority class resource, which we already created along with the other priority classes very early during cluster setup.

It appears that the Postgres Operator Helm chart is trying to create the **neon-data** priority class even though I've previously commented out the contents of the `postgres-pod-priority-class.yaml` template file.  I just tried removing that file and we still see this problem.
